### ISSUE TITLE
Organization received a new IP pool config, but the IP used is the old one

### ISSUE THREAD

**Issue by franciscopaniskaseker** [postalserver/postal#2034]
## Describe the bug

Organization received a new IP pool config, but the IP did not change.

## To Reproduce

1. Set the organization X to use a pool Y (using web gui).
2. Change the pool Y to another name (using web gui).
3. Change the pool Z name (using web gui).
4. Set the organization X to use the pool Z (using web gui).
5. Try to send an e-mail. The organization X is still using the pool Y (we can see the old IP).

## Expected behaviour

The organization X use the new pool (Z) IP.

## Screenshots

no

## Environment details 

 - OS: Ubuntu 20.04 and docker setup
 - Browser: chrome and safari
 - Version: Postal 2.1.1

## Additional information/context

The email traffic is ok and all IPs are working. We just can not change the ip pool in this way. We did not try remove the ip pool and create again.

**Comment by willpower232** [comment]
Postal can only bind to IPs present on the system, can you definitely see the IP from the IP Pool with the command `ip -a` on the server?

I'm not super familiar with them so the only other thing I could suggest is restarting just to be sure everything has applied properly.

**Comment by franciscopaniskaseker** [comment]
Yes, the IP is there. Actually both IPs were being used before the change. Postal can see them and can use them (old configs). This bug happened when I tried to swap the IPs with a new names.

**Comment by franciscopaniskaseker** [comment]
How can I list and maybe set through the console? i am not finding any documentation about how to use the console. Am I missing something?
